ui/admin/package.json

```diff
@@ -63,6 +63,7 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-template-lint": "^2.0.2",
     "ember-cli-terser": "^4.0.2",
+    "ember-composable-helpers": "^4.5.0",
```

Comment:
Nice! We are not seeing a change in `yarn.lock` because we already add `ember-composable-helpers: ^4.5.0` So all good!